src/views/orders.vue

```diff
@@ -277,8 +278,20 @@ export default defineComponent({
       selectedItemsCount: 'order/getSelectedItemsCount',
       userProfile: 'user/getUserProfile',
       query: 'order/getQuery',
+      expirationTime: 'user/getExpirationTime'
     }),
   },
+  created() {
+    const currentTime = new Date().getTime();
+    if(this.expirationTime <= currentTime) {
+      this.isTokenExpired = true;
+    }
+  },
+  mounted() {
+    if(this.isTokenExpired) {
```

Comment:
I think this will work only for orders page. We need a generic solution that works across the app

Reply:
Sure sir.